File: BKabanApi/BKabanApi/Models/UserModel.cs
using System.ComponentModel.DataAnnotations;

namespace BKabanApi.Models
{
    public class UserModel
    {
        [Required(ErrorMessage = "Username required")]
        [MaxLength(20, ErrorMessage = "Max username length is {1}")]
        [MinLength(2, ErrorMessage = "Min username length is {1}")]
        public string Username { get; set; }

        [Required(ErrorMessage = "Password required")]
        [MaxLength(100, ErrorMessage = "Max password length is {1}")]
        [MinLength(6, ErrorMessage = "Min password length is {1}")]
        public string Password { get; set; }
    }
}

File: BKabanApiJava/src/main/java/edu/bkaban/models/UserModel.java
package edu.bkaban.models;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

public class UserModel {

    @NotNull(message = "Username required")
    @Size(min = 2, max = 20, message = "Username length have to be between 2 and 20")
    private String Username;

    @NotNull(message = "Password required")
    @Size(min = 6, max = 100, message = "Password length have to be between 6 and 100")
    private String Password;

    public String getUsername() {
        return Username;
    }

    public void setUsername(String username) {
        Username = username;
    }

    public String getPassword() {
        return Password;
    }

    public void setPassword(String password) {
        Password = password;
    }

    public UserModel(String username, String password) {
        Username = username;
        Password = password;
    }

    public UserModel() {}
}
